docs/user_guide.md

```diff
@@ -34,14 +34,18 @@ Setup your Python environment with the following commands:
 # Set up a virtual environment to isolate packages from other envs.
 python3.11 -m venv 3.11.venv
 source 3.11.venv/bin/activate
+```
+
+## Install SHARK and its dependencies
+
+Before installing `shark-ai` install a torch version that fulfills your needs.
 
-# Optional: faster installation of torch with just CPU support.
+```bash
+# Fast installation of torch with just CPU support.
 # See other options at https://pytorch.org/get-started/locally/
```

Comment:
I guess we could pull this URL out of the code block so it is clickable 🤔

Reply:
Sounds good.